### ISSUE TITLE
Strange behavior with Polymer JS app

### ISSUE THREAD

**Issue by kaspnilsson** [sourcegraph/sourcegraph#511]
- **Sourcegraph version:** 
1.17.0
- **Platform information:**
Chrome 69.0.3497.81 (Official Build) (64-bit)
MacOS

#### Steps to reproduce:

1. Visit https://github.com/GerritCodeReview/gerrit/blob/master/polygerrit-ui/app/elements/gr-app.js
2. Mouse over the type definition on line 38

#### Expected behavior:
Not an error...?

#### Actual behavior:
"Debug Failure. False expression." in the tooltip. From network console:
```
    "error": {
      "code": -32001,
      "message": "Debug Failure. False expression.",
      "data": {}
    },
```

**Comment by sqs** [comment]
Assigned to @felixfbecker to see if this still occurs in the new TypeScript extension in 3.0-preview and fix it if it does (or defer to milestone 3.0 if not possible to fix).

**Comment by felixfbecker** [comment]
Works for me with the new extension:

![2018-11-15 11 20 07](https://user-images.githubusercontent.com/10532611/48546357-74792600-e8c8-11e8-899b-a7953f0bc194.gif)

But I also cannot reproduce with the current TypeScript support:

![2018-11-15 11 20 57](https://user-images.githubusercontent.com/10532611/48546413-98d50280-e8c8-11e8-831e-88025bc35a27.gif)

I think this was a bug in the TypeScript compiler that was fixed in upgrading TypeScript at some point.
